File: 网课/中国大学MooC/哈佛CS50/Lecture 4/struct0.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

#include "struct.h"

int main(void){
    //输入多少学生
    int enrollment=get_int("Enrollment:");
    // 调用头文件创造的数据类型
    student students[enrollment];
    //输入
    for(int i=0;i<enrollment;i++){
        students[i].name=get_string("Name:");
        students[i].dorm=get_string("Dorm:");
    }
    //输出
    for(int i=0;i<enrollment;i++){
        printf("%s is in %s\n",students[i].name,students[i].dorm);
    }
}

File: 网课/哈佛CS50/Lecture 6/struct0.py
from cs50 import get_string

students=[]

for i in range(3):
    name=get_string("name:")
    dorm=get_string("dorm:")

    student={"name":name,"dorm":dorm}

    students.append(student)

for student in students:
    print(f"{student['name']} 在{student['dorm']}")